cli/rt/22_form_data.js

```diff
@@ -1,21 +1,19 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
 ((window) => {
-  const blob = window.__bootstrap.blob;
-  const domFile = window.__bootstrap.domFile;
```

Comment:
I've changed my mind, I don't like this change - now this file implictly depends on both `Blob` and `File` - I much more prefer that those "deps" are listed at the top of the file.

Reply:
@ry we should hold of this change until we actually factor out op crates. Right now it's just unnecessarily complicating the setup by making dependencies implicit.

Reply:
ok fair enough. I will back out the controversial parts of this patch.